packages/poaps/src/types/input.ts

```diff
@@ -17,3 +17,10 @@ export interface FetchPoapsInput extends PaginationInput {
   sort_dir?: Order;
   filter_by_zero_address?: boolean;
 }
+
+export interface claimtInput {
```

Comment:
Why this one is `pascalCase` instead of `CamelCase`?

It's all switched. Super weird.